services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -184,12 +184,12 @@ export class NoticeOfWorkApplication extends Component {
         <div className={this.state.fixedTop ? "steps--header fixed-scroll" : "steps--header"}>
           <div className="inline-flex between">
             <div>
-              <h1>NoW Number: {Strings.EMPTY_FIELD}</h1>
+              <h1>NoW Number: {this.props.noticeOfWork.now_number || Strings.EMPTY_FIELD}</h1>
               {/* update to use application_guid for link once guid is persisted */}
```

Comment:
I think this entire comment block/following <Link> can be removed now, it used to link to the original NOW.

Reply:
Removed :)